contracts/src/Gateway.sol

```diff
@@ -97,7 +97,7 @@ contract Gateway is IGateway, IInitializable {
     }
 
     constructor(
-        address beefyClient,
+        address beefyLightClient,
```

Comment:
I specifically dropped the "Light" qualifier. It doesn't add anything meaningful, and is too verbose.

Reply:
The name shadowed the `beefyClient()` method. I removed the word light and used an underscore to avoid the name collision.